editor/src/clj/prof.clj

```diff
@@ -24,10 +24,10 @@
   (future
     (try
       (.browse (Desktop/getDesktop) uri)
-      nil
       (catch Throwable error
         (println (str "Failed to open URI " uri))
-        (println (.getMessage error))))))
+        (println (.getMessage error)))))
+  nil)
```

Comment:
This file contains profiling functions that are never called by the editor. This function was erroneously returning the `future` it created. Fixed to return `nil`.